### PR TITLE
enh: add project links

### PR DESCRIPTION
**Pull request by evenstensberg**
Closes https://github.com/openjs-foundation/cross-project-council/issues/215 and fixes some whitespace

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,45 +12,45 @@ The CPC exercises autonomy in managing its responsibilities and seeks agreement
...
 
 #### Emeritus Projects
 
-* Chassis
-* jQuery Mobile
-* jQuery UI
-* RequireJS
-* Sizzle
+* [Chassis](https://github.com/jquery/css-chassis)
+* [jQuery Mobile](http://jquerymobile.com/)
```
Comment (molant) [inline README.md]:
```suggestion
* [jQuery Mobile](https://jquerymobile.com/)
```

Doesn't redirect

File: README.md [hunk 2]
```diff
@@ -12,45 +12,45 @@ The CPC exercises autonomy in managing its responsibilities and seeks agreement
...
+* [jQuery UI](http://jqueryui.com/)
```
Comment (molant) [inline README.md]:
```suggestion
* [jQuery UI](https://jqueryui.com/)
```
Doesn't redirect

File: README.md [hunk 3]
```diff
@@ -12,45 +12,45 @@ The CPC exercises autonomy in managing its responsibilities and seeks agreement
...
+* [RequireJS](http://requirejs.org/)
```
Comment (molant) [inline README.md]:
```suggestion
* [RequireJS](https://requirejs.org/)
```

**Comment by MylesBorins** [comment]
@molant thank you so much for doing this review... this is great information that we can use to get the landing pages for projects following best practices

@openjs-foundation/cpc how would you like to process with this? Should we make a project board? Is this something we want to request @brianwarner or the foundation to help make sure we follow up on?

**Comment by kborchers** [comment]
I went through and added redirects to the ones I could but I didn't have access to some. I would say this is something that should be followed up on to make sure the foundation has access to and the ability to manage domain registrations, DNS, SSL certs, etc for all project domains to ensure they stay up and running and that burden is taken off of the project maintainers.

**Comment by evenstensberg** [comment]
Updated the ones mentioned. LMK if there's more left todo

File: README.md [hunk 4]
```diff
@@ -12,45 +12,45 @@ The CPC exercises autonomy in managing its responsibilities and seeks agreement
 
 #### Impact Project
 
-* Appium
-* Dojo
-* jQuery
-* Node.js
-* webpack
+* [Appium](http://appium.io/)
```
Comment (trivikr) [inline README.md]: Appium seems to have valid HTTPS cert https://appium.io/
```suggestion
* [Appium](https://appium.io/)
```

File: README.md [hunk 5]
```diff
@@ -12,45 +12,45 @@ The CPC exercises autonomy in managing its responsibilities and seeks agreement
...
-* JerryScript
-* Libuv
-* Lodash
-* Marko
-* messageformat
-* Moment
-* PEP
-* QUnit
+* [ESLint](http://eslint.org/)
+* [Esprima](http://esprima.org/)
```
[the remark this answers is not in the document]
Comment (trivikr) [reply README.md]: Created issue at https://github.com/jquery/esprima/issues/1975

File: README.md [hunk 6]
```diff
@@ -12,45 +12,45 @@ The CPC exercises autonomy in managing its responsibilities and seeks agreement
...
+* [Express](https://expressjs.com/)
+* [Grunt](http://gruntjs.com/)
+* [HospitalRun](http://hospitalrun.io/)
+* [Interledger.js](http://interledgerjs.org/)
+* [JerryScript](http://jerryscript.net/)
```
[the remark this answers is not in the document]
Comment (trivikr) [reply README.md]: Created issue at https://github.com/jerryscript-project/jerryscript/issues/2924

**Comment by MylesBorins** [comment]
been more than 72 hours, landing

**Pull request merged**